extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -1066,7 +1069,7 @@ export class AmpLightboxGallery extends AMP.BaseElement {
   enter_() { // TODO (cathyxz): make this generalizable to more than just images
     const {sourceElement} = this.getCurrentElement_();
     if (!this.elementTypeCanBeAnimated_(sourceElement)) {
-      return this.fade_(0, 1);
+      return this.fade_(true);
```

Comment:
Can we have a enum (transition = ENTER | EXIT) for this? Or a comment: `fade(/*fadeIn*/true)`.

Reply:
Went for a comment since a enum seems a bit overkill.